zstacklib/zstacklib/utils/shared_cpuset.py

```
@@ -0,0 +1,364 @@
+            raise SharedCpuSetError('PID_START_TIME_UNAVAILABLE')
+        try:
+            start_ticks = int(fields[19])
+            clock_ticks = os.sysconf('SC_CLK_TCK')
+            if clock_ticks <= 0:
+                raise SharedCpuSetError('PID_START_TIME_UNAVAILABLE')
+            return boot_time + float(start_ticks) / clock_ticks
+        except (TypeError, ValueError, OSError, ZeroDivisionError):
+            raise SharedCpuSetError('PID_START_TIME_UNAVAILABLE')
+
+    def _systemd_properties(self, unit):
+        process = subprocess.Popen([
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)